docs/running-wordpress-on-kubernetes.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Running WordPress on Kubernetes
+linktitle: Running WordPress on Kubernetes
+description: "There are multiple parts that make a site running on Kubernetes via Stack, and we'll take them one by one below."
+categories: []
+keywords: ['stack', 'docs', 'wordpress', 'kubernetes']
+draft: false
+aliases: []
+slug: running-wordpress-on-kubernetes
+toc: true
+related: true
+---
+
+A good reference about how a WordPress site looks on Stack is the [WordPress Spec](https://github.com/presslabs/wordpress-operator#deploying-a-wordpress-site).
+
+## Domains
+
+Each site needs to have at least one domain. When a request comes to the NGINX Ingress, it'll get routed to the appropriate pods, based on the `Host` header.
+
+Even if you can have multiple domains answering to the same site, you still need the main domain that will be responsible for the `WP_HOME` and `WP_SITEURL` constants.
+
+Those domains are syncing in the ingress controller. Also, Cert Manager will bundle those domains into one single certificate.
```

Comment:
```suggestion
Those domains are syncing in the ingress controller. Also, [cert-manager](https://github.com/jetstack/cert-manager) will bundle those domains into one single certificate.
```
That appears to be the spelling of the project's name: https://github.com/jetstack/cert-manager#cert-manager
If it's not obvious enough it's a name some options would be to use an URL as suggested, or maybe use  code marks : `cert-manager`.